client/lcd/root.go

```diff
@@ -129,7 +130,7 @@ func (rs *RestServer) Start(listenAddr string, sslHosts string,
 
 		err := rpcserver.StartHTTPAndTLSServer(
 			rs.listener,
-			rs.Mux,
+			cors.Default().Handler(rs.Mux),
```

Comment:
Should we consider the `cors` flag or always enable CORS?

Reply:
++

Reply:
A flag sounds prudent; some users might not want CORS enabled.